src/main/java/bizbook/ui/CommandBox.java

```diff
@@ -43,11 +47,37 @@ private void handleCommandEntered() {
         try {
             commandExecutor.execute(commandText);
             commandTextField.setText("");
+            commandHistory.addCommand(commandText);
         } catch (CommandException | ParseException e) {
             setStyleToIndicateCommandFailure();
         }
     }
 
+    /**
+     * Populates command box with  {@param text}.
```

Comment:
Additional whitespace.
```suggestion
     * Populates command box with {@param text}.
```